docs/source/workflow_inputs.rst

```diff
@@ -370,7 +370,7 @@ Building occupancy is entered in ``/HPXML/Building/BuildingDetails/BuildingSumma
   ========================================  ========  =====  ===========  ========  ====================  ========================
   Element                                   Type      Units  Constraints  Required  Default               Notes
   ========================================  ========  =====  ===========  ========  ====================  ========================
-  ``NumberofResidents``                     integer          >= 0         See [#]_  <number of bedrooms>  Number of occupants [#]_
+  ``NumberofResidents``                     double           >= 0         See [#]_  <number of bedrooms>  Number of occupants [#]_
```

Comment:
Why double?

Reply:
Because the documentation was wrong.

- https://hpxml.nrel.gov/datadictionary/3.0.0/Building/BuildingDetails/BuildingSummary/BuildingOccupancy/NumberofResidents
- https://github.com/NREL/OpenStudio-HPXML/blob/ac525ddca5d5178e089bb1104e86edc59ddbde4e/HPXMLtoOpenStudio/resources/hpxml.rb#L1284

Does the stochastic generator support a non-integer? If not, it should probably round to the nearest integer.

Reply:
@rajeee Do you know if the stochastic generator supports a non-integer number of occupants?